core/modules/node/views/views_handler_filter_history_user_timestamp.inc

```diff
@@ -55,9 +55,8 @@ class views_handler_filter_history_user_timestamp extends views_handler_filter {
       return;
     }
 
-    // Hey, Backdrop kills old history, so nodes that haven't been updated
-    // since NODE_NEW_LIMIT are bzzzzzzzt outta here!
```

Comment:
🤣 ...we should change the language in all comments to this!

Reply:
:stuck_out_tongue_closed_eyes: